docusaurus.config.js

```diff
@@ -10,7 +10,13 @@ module.exports = {
   favicon: 'img/favicon.ico',
   organizationName: 'NEAR',
   projectName: 'sdk-docs',
+  themes: [
+    '@saucelabs/theme-github-codeblock'
+  ],
   themeConfig: {
+    prism: {
+      additionalLanguages: ['rust'],
+    },
```

Comment:
I wonder if this solved https://github.com/near/sdk-docs/issues/23 or not. Couldn't tell since the render bot never generated another render after the initial failure